src/app/compliance-rule-configuration/configure-compliance-rule/configure-compliance-rule.component.ts

```diff
@@ -74,39 +79,32 @@ export class ConfigureComplianceRuleComponent implements OnInit {
     return this.complianceRuleParameters.filter(param => param.name.includes(assignmentName))[0]
   }
 
-
-
   emitComplianceRuleConfiguration( conf : EntityModelComplianceRuleConfigurationEntity) {
     this.complianceRuleConfigurationEmitter.emit(conf);
   }
 
   storeComplianceRuleConfiguration() {
```

Comment:
I think you should also allow changing the issue type here (not only provide assignments)

Reply:
done